lblod/frontend-lokaal-mandatenbeheer
# automatically create bestuursorgaan and mandates from template

## Description

- auto create mandates for bestuursorgaan
- use the proper, new conceptscheme for non standard bestuursorgaan instances
- have the forms wait on their onSave and onCreate promise so they don't seem unresponsive

## How to test

Go to the interface to create new bestuursorgaan, use it, see that the mandates have automatically been added.

## Links to other PR's

- https://github.com/lblod/app-lokaal-mandatenbeheer/pull/228
